From 54889436af8785958d1efdb79730bf662afc5678 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Sun, 8 Nov 2020 14:40:27 +0800
Subject: [PATCH] drm/rockchip: vop2: Fix smatch warning

drivers/gpu/drm/rockchip/rockchip_drm_vop2.c:3499
vop2_plane_create_name_property() warn: should '1 << (win->win_id)'
be a 64 bit type?

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: If4d46563519dabb18792c27bf4069f037de20879
---
 drivers/gpu/drm/rockchip/rockchip_drm_vop2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rockchip/rockchip_drm_vop2.c b/drivers/gpu/drm/rockchip/rockchip_drm_vop2.c
index a5bdbe4ff768..15c03cde5377 100644
--- a/drivers/gpu/drm/rockchip/rockchip_drm_vop2.c
+++ b/drivers/gpu/drm/rockchip/rockchip_drm_vop2.c
@@ -3496,7 +3496,7 @@ static int vop2_plane_create_name_property(struct vop2 *vop2, struct vop2_win *w
 {
 	struct drm_prop_enum_list *props = vop2->plane_name_list;
 	struct drm_property *prop;
-	uint64_t bits = BIT(win->win_id);
+	uint64_t bits = BIT_ULL(win->win_id);
 
 	prop = drm_property_create_bitmask(vop2->drm_dev,
 					   DRM_MODE_PROP_IMMUTABLE, "NAME",
-- 
2.35.3

